coding/pb/ipld.pb.go.patch

```diff
@@ -0,0 +1,28 @@
+--- ipld.pb.go.old	2016-02-24 10:29:39.045261549 +0100
++++ ipld.pb.go.new	2016-02-24 10:29:45.038594961 +0100
```

Comment:
See gogo/protobuf#151 (the field order is wrong when regenerating the protocol buffer code)